iceoryx_posh/test/moduletests/test_popo_chunk_sender.cpp

```diff
@@ -105,24 +110,28 @@ class ChunkSender_test : public Test
 
 TEST_F(ChunkSender_test, allocate_OneChunk)
 {
-    auto maybeChunkHeader = m_chunkSender.tryAllocate(sizeof(DummySample), iox::UniquePortId());
+    auto maybeChunkHeader = m_chunkSender.tryAllocate(
```

Comment:
In theory it could be possible that the custom header arguments are just ignored inside of `tryAllocate`. Therefore I would add a test here that verifies that `tryAllocate` with a custom header size != `iox::CHUNK_NO_CUSTOM_HEADER_SIZE` works and provides a chunk with the correct size/alignment